src/foam/nanos/auth/PasswordHashingDAO.java

```diff
@@ -22,11 +26,53 @@ public PasswordHashingDAO(X x, DAO delegate) {
   }
 
   @Override
+  public Sink select_(X x, Sink sink, long skip, long limit, Comparator order, Predicate predicate) {
+    if ( sink == null ) sink = new ListSink();
+    ProxySink proxy = new ProxySink(x, sink) {
```

Comment:
Make own top-level class rather than in-line sub-class so that it's serializable for network DAO's.

Reply:
Okay done.